portfolio/pom.xml

```diff
@@ -41,4 +41,4 @@
       </plugin>
     </plugins>
   </build>
-</project>
```

Comment:
Please fix -- "No newline at end of file"

Reply:
Thanks, fixed.